File: app/models/suggestion.rb
class Suggestion < ActiveRecord::Base
  attr_accessible :concept, :description, :user_id, :organization_id, :votes_count

  belongs_to :organization, :counter_cache => true
  belongs_to :user
  has_many :votes

  validates :concept, :presence => true
  validates :description, :presence => true

  def as_json(object)
    { 
      concept: concept, description: description, 
      user_name: user.name,
      gravatar_id: user.gravatar_id
    }
  end

end

File: app/assets/javascripts/backbone/models/suggestion.js
(function(Backbone) {

  TalkThatTalk.Models.Suggestion = Backbone.Model.extend({
    defaults: {
      concept: '',
      description: '',
      user_name: '',
      gravatar_id: '',
      created_at: null,
      votes_count: 0
    }
  });

}(Backbone));